specification/sqlvirtualmachine/resource-manager/readme.md

```diff
@@ -49,6 +49,24 @@ input-file:
 - Microsoft.SqlVirtualMachine/preview/2017-03-01-preview/sqlvm.json
 ```
 
+### Tag: package-2022-02-01-preview
```

Comment:
let's move the 2022 tags to above the 2021 tags, rather than placing them below the 2017 tags